docs/reference/docs/termvectors.asciidoc

```diff
@@ -86,6 +86,40 @@ or the field statistics of the entire index, and not just at the shard. Use it
 with caution as distributed frequencies can have a serious performance impact.
 
 [float]
+==== Terms Filtering coming[2.0]
+
+With the parameter `filter`, the terms returned could also be filtered based
```

Comment:
One nice thing about using `filter` as a sub-level parameter is that `filter : {}` with no parameter will perform filtering but using the default sub parameters. Another reason would be to hide the complexity of a feature which might be used only in some very specific cases. It also allows us to expand on this sub feature, in order, for example, to allow filtering based on a script more cleanly I think. But I do agree, I am not sure if `filter` should be the proper name for this feature.